lib/inputs/runtime/docker.ts

```diff
@@ -0,0 +1,15 @@
+import { Docker, DockerOptions } from "../../docker";
+
+export function getRuntime(options: DockerOptions) {
+  return Docker.run(["version"], options)
+    .then((output) => {
```

Comment:
why not using `async/await` instead?

Reply:
I only moved the file, I haven't optimised it or changed it to something different 😄

Reply:
it's not necessary but we can give it some "runtime touch" 😺